app/controllers/application_controller.rb

```diff
@@ -1,10 +1,143 @@
 require './config/environment'
+require 'pry'
 
 class ApplicationController < Sinatra::Base
 
   configure do
     set :public_folder, 'public'
     set :views, 'app/views'
+    enable :sessions
+    set :session_secret, "password_security"
+  end
+
+  get '/' do
+  	erb :index
+  end
+
+  get '/tweets' do
+  	if logged_in?
+  		@tweets = Tweet.all
+  		erb :'tweets/tweets'
+  	else
+  		redirect '/login'
+  	end
+  end
+
+  get '/tweets/new' do
+  	if logged_in?
+  		erb :'/tweets/create_tweet'
+  	else
+  		redirect '/login'
+  	end
+  end
+
+  post '/tweets' do
+  	if params[:content] != ""
+	  	@tweet = Tweet.new(params)
+	  	@tweet.user = current_user
```

Comment:
Another way to do this is to make use of the `has_many` relationship on User:
```
@tweet = current_user.tweets.create(content: params[:content])
```
Both approaches are good, but in some cases you may find this option useful.